crates/nu-command/tests/commands/default.rs

```diff
@@ -24,7 +24,7 @@ fn adds_row_data_if_column_missing() {
             r#"
                 open los_tres_amigos.json
                 | get amigos
-                | default rusty_luck 1
+                | default 1 rusty_luck
```

Comment:
By the way… am I losing my mind, or was this test always wrong until now? `default rusty_luck 1` is not the correct order of `default` arguments, and never was, right?

Reply:
Suspect the test was always wrong.

FWIW I also assumed that the order was `default <column name> <default value>` instead of when I first encountered `default`, I find the current order somewhat unintuitive.